### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by abdelp**
This pull request contains all of the necessary files to complete the requirements listed [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896082/)

- Display mode and width for App, Display, and ButtonPanel componets.
- Styling for Display, ButtonPanel, and Buttons

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import PropType from 'prop-types';
 
-const Button = ({ buttonName }) => (
-  <button type="button">{buttonName}</button>
+const Button = ({ buttonName, color = 'orange' }) => (
```
Comment (suretrust) [inline src/components/Button.js]: - [x] You shouldn't set the value of `color` directly as such. Use `defaultProps` instead. You can read about it [here](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values).

File: src/components/Display.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropType from 'prop-types';
 
 const Display = ({ result = '0' }) => (
```
Comment (suretrust) [inline src/components/Display.js]: Here too.

File: src/styles/scss/App.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    .btn {
+      width: 25%;
+      height: 100%;
+      border: 1px solid black;
+      font-size: large;
+    }
+
+    .btn-group:last-child .btn:first-child {
+      width: 50%;
+    }
```
Comment (suretrust) [inline src/styles/scss/App.scss]: - [x] This should be set using `props`, not directly in the CSS.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @abdelp, 👋 

### Great job thus far! 🙌 Please see my comments. 👇 

- [x] Add props validation to the necessary components.

* [x] The `Button` component should accept `color` and `wide` as an incoming props ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png))

* [x] `wide` is boolean, true means that button take 50% of space (used for `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png))

**KIndly submit another CRR once these issues have been solved.** _Happy coding!_ 🎉

File: src/components/Button.js (hunk 1 above)
Comment (abdelp) [reply src/components/Button.js]: changed

File: src/styles/scss/App.scss (hunk 3 above)
Comment (abdelp) [reply src/styles/scss/App.scss]: logic moved to the component

File: src/components/Button.js (hunk 1 above)
Comment (abdelp) [reply src/components/Button.js]: Refactored

**Review by bellom — APPROVED** [review]
## Hi @abdelp  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**